Routes needs new method Routes.Find()

Compiling hostrouter currently seems to need the addition of a Routes.Find( method to comply with the chi.Routes interface.:

../../../../go/pkg/mod/github.com/go-chi/hostrouter@v0.2.0/hostrouter.go:12:20: cannot use Routes{} (value of type Routes) as chi.Routes value in variable declaration: Routes does not implement chi.Routes (missing method Find)

--- a/go.mod
+++ b/go.mod
@@ -2,4 +2,4 @@ module github.com/go-chi/hostrouter
 
 go 1.16
 
-require github.com/go-chi/chi/v5 v5.0.0
+require github.com/go-chi/chi/v5 v5.2.0

--- a/hostrouter.go
+++ b/hostrouter.go
@@ -19,6 +19,10 @@ func (hr Routes) Match(rctx *chi.Context, method, path string) bool {
 	return true
 }
 
+func (hr Routes) Find(rctx *chi.Context, method, path string) string {
+	return ""
+}
+
 func (hr Routes) Map(host string, h chi.Router) {
 	hr[strings.ToLower(host)] = h
 }

